buildspec.json

```diff
@@ -1,88 +1,67 @@
+            "version": "2023-04-12",
             "baseUrl": "https://github.com/obsproject/obs-deps/releases/download",
-            "label": "Pre-built Qt6",
+            "label": "Pre-Built Qt6",
             "hashes": {
-                "macos-x86_64": "a83f72a11023b03b6cb2dc365f0a66ad9df31163bbb4fe2df32d601856a9fad3",
-                "macos-arm64": "2f30af90c049670a5660656adbb440668aa1b0567f75a5f29e1def9108928403",
-                "macos-universal": "252e6684f43ab9c6f262c73af739e2296ce391b998da2c4ee04c254aaa07db18",
-                "windows-x64": "e5509b54196a3f935250cc4b9c54160c8e588fd0f92bc078a2a64f9d9e2e4e93",
-                "windows-x86": "24fc03bef153a0e027c1479e42eb08097a4ea1d70a4710825be0783d0626cb0d"
+                "macos": "eb7614544ab4f3d2c6052c797635602280ca5b028a6b987523d8484222ce45d1",
+                "windows-x64": "4d39364b8a8dee5aa24fcebd8440d5c22bb4551c6b440ffeacce7d61f2ed1add"
             },
-            "pdb-hashes": {
-                "windows-x64": "60e5b1d2bc4d7c431bc05f14e3b1e85e088788c372fa85f58717cd6c49555a46",
-                "windows-x86": "f34d1a89fc85d92913bd6c7f75ec5c28471d74db708c98161100bc8b75f8fc63"
+            "debugSymbols": {
+                "windows-x64": "f34ee5067be19ed370268b15c53684b7b8aaa867dc800b68931df905d679e31f"
             }
         }
     },
+    "tools": {
+        "packages": {
+            "versions": "1.2.10",
+            "baseUrl": "http://s.sudre.free.fr/Software/files",
+            "label": "Packages.app",
+            "hash": "6afdd25386295974dad8f078b8f1e41cabebd08e72d970bf92f707c7e48b16c9"
+        }
+    },
     "platformConfig": {
-        "macos-x86_64": {
-            "qtVersion": 6,
-            "deploymentTarget": "10.15"
-        },
-        "macos-arm64": {
+        "macos": {
             "qtVersion": 6,
-            "deploymentTarget": "11.0"
-        },
-        "macos-universal": {
-            "qtVersion": 6,
-            "deploymentTarget": "10.15"
+            "deploymentTarget": "11.0",
+            "bundleId": "com.example.obs-plugintemplate"
         },
         "windows-x64": {
             "qtVersion": 6,
             "visualStudio": "Visual Studio 17 2022",
             "platformSDK": "10.0.20348.0"
         },
-        "windows-x86": {
-            "qtVersion": 6,
-            "visualStudio": "Visual Studio 17 2022",
-            "platformSDK": "10.0.20348.0"
-        },
         "linux-x86_64": {
             "qtVersion": 6
-        },
-        "linux-aarch64": {
-            "qtVersion": 6
         }
     },
     "name": "obs-plugintemplate",
     "version": "1.0.0",
     "author": "Your Name Here",
-    "email": "me@mymailhost.com"
+    "website": "https://example.com",
+    "email": "me@example.com",
+    "uuids": {
+        "macosPackage": "0B7A74BC-65CF-4FF1-AC34-5C743E8A48F5",
+        "macosInstaller": "52B6084A-F58D-45C3-BE37-76AD45F16072",
+        "windowsApp": "CD703FE5-1F2C-4837-BD3D-DD840D83C3E3"
```

Comment:
Are these UUIDs meant to be generated by the plugin author?
It might make sense to replace them with `XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX` or something invalid and throwing a warning if it's not changed, I can envision many plugin authors not changing them otherwise (because they might not realize).

Reply:
It would make sense to replace this with a placeholder (which needs replacing) but it also needs to be valid, otherwise CI on this repo would fail.

But if we use a valid value here, there's the risk of plugins re-using the same UUID.

Reply:
The only thing I can think of is to provide a valid UUID that is obviously "wrong" looking (all zeroes or some such), and writing some kind of check that will allow this on obsproject/obs-plugintemplate CI, but warn/error/fail on any other CI and give instructions on how to generate replacement UUIDs.

Reply:
That wouldn't prohibit their "bad" UUIDs from also work on their own CI runs, except if we _also_ add checks if its our repo and at that point I'd question if that is all worth the effort.

Using a UUID entirely of `0` or `F` seems good enough as a possible alternative, because at some point developers just need to know what they're doing.

Reply:
> except if we _also_ add checks if its our repo and at that point I'd question if that is all worth the effort.

That is more or less what I was describing, yes.

Reply:
> Using a UUID entirely of `0` or `F` seems good enough as a possible alternative, because at some point developers just need to know what they're doing.

Apologies for only responding now (I was a bit busy this evening), but yes, this was my idea behind suggesting `XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX` (or which ever character you want to use), it’s something that much more obviously needs changing than an already set "normal" UUID.